charts/nexus-iq/values.yaml

```diff
@@ -1,6 +1,9 @@
 # Default values for iqserver.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+statefulset:
+  # This is not supported
```

Comment:
does this mean that we don't support it, or that by setting `enabled: false` we won't use the (supported) feature? Maybe 

> # disable statefulset so that blah blah

Reply:
this makes it consistent with the nxrm helm chart, but unfortunately, it doesn't really explain the comment. 🤔 maybe we don't need to mirror *everything* from the nxrm helm chart if it's strangely defined.

Reply:
I'm OK with this pass just getting consistent though. we'll eventually need to pass through all the params to validate their functionality.

Reply:
👍